src/Maker/MakeResetPassword.php

```diff
@@ -0,0 +1,277 @@
+
+        $controllerClassNameDetails = $generator->createClassNameDetails(
+            'ResetPasswordController',
+            'Controller\\'
+        );
+
+        $requestClassNameDetails = $generator->createClassNameDetails(
+            'ResetPasswordRequest',
+            'Entity\\'
+        );
+
+        $repositoryClassNameDetails = $generator->createClassNameDetails(
+            'ResetPasswordRequestRepository',
+            'Repository\\'
+        );
+
+        $requestFormTypeClassNameDetails = $generator->createClassNameDetails(
+            'ResetPasswordRequestFormType',
+            'Form\\'
+        );
+
+        $changePasswordFormTypeClassNameDetails = $generator->createClassNameDetails(
+            'ChangePasswordFormType',
+            'Form\\'
+        );
+
+        $generator->generateController(
+            $controllerClassNameDetails->getFullName(),
+            'resetPassword/ResetPasswordController.tpl.php',
+            [
+                'user_full_class_name' => $userClassNameDetails->getFullName(),
+                'user_class_name' => $userClassNameDetails->getShortName(),
+                'request_form_type_full_class_name' => $requestFormTypeClassNameDetails->getFullName(),
+                'request_form_type_class_name' => $requestFormTypeClassNameDetails->getShortName(),
+                'reset_form_type_full_class_name' => $changePasswordFormTypeClassNameDetails->getFullName(),
+                'reset_form_type_class_name' => $changePasswordFormTypeClassNameDetails->getShortName(),
+                'password_setter' => $input->getArgument('password-setter'),
+                'success_redirect_route' => $input->getArgument('controller-reset-success-redirect'),
+                'from_email' => $input->getArgument('from-email-address'),
+                'from_email_name' => $input->getArgument('from-email-name'),
+                'email_getter' => $input->getArgument('email-getter'),
+            ]
+        );
+
+        $generator->generateClass(
+            $requestClassNameDetails->getFullName(),
+            'resetPassword/ResetPasswordRequest.tpl.php',
+            [
+                'repository_class_name' => $repositoryClassNameDetails->getFullName(),
+                'user_full_class_name' => $userClassNameDetails->getFullName(),
+            ]
+        );
+
+        $generator->generateClass(
+            $repositoryClassNameDetails->getFullName(),
+            'resetPassword/ResetPasswordRequestRepository.tpl.php',
+            [
+                'request_class_full_name' => $requestClassNameDetails->getFullName(),
+                'request_class_name' => $requestClassNameDetails->getShortName(),
+            ]
+        );
+
+        if (!$this->fileManager->fileExists($path = 'config/packages/reset_password.yaml')) {
+            throw new RuntimeCommandException(sprintf('The file "%s" does not exist. This command needs that file to accurately build the reset password config.', $path));
```

Comment:
Per my comment below, this is not true. It IS unexpected for sure. I guess in this case I would... still create the file from "scratch".

Reply:
logic removed - see `MakeResetPassword::setBundleConfig()`